src/libs/ajax/fetch/fetchLeoOnce.ts

```diff
@@ -0,0 +1,5 @@
+// For Leo calls which should not trigger signout on auth failure.
+import { fetchOk, withUrlPrefix } from 'src/libs/ajax/fetch/fetch-core';
+import { getConfig } from 'src/libs/config';
+
+export const fetchLeoOnce = withUrlPrefix(`${getConfig().leoUrlRoot}/`, fetchOk);
```

Comment:
hmmm.... fetchLeoOnce doesn't seem to need anything from auth-fetch.ts where this PR has the larger impact of moving authOpts and withAuthSession to a separate auth-options,ts module.

I'd suggest not making that split if it is not required... would greatly reduce the PR impact.

Reply:
I'll combine these two PRs, as the changes don't make sense split up.

Reply:
ok yeah, combining was gonna be my next thought so we can see the linkage

Reply:
I meant to suggest earlier a different name for fetchLeoOnce.... since it doesn't necessarily need the "Once" assumption, but it is a non-auth fetch... so how about fetchLeoPublic instead?

Reply:
@msilva-broad I went with the old `fetchLeoWithoutAuthRetry` - wordy, but this call is made with auth, just doesn't retry on failure.